migrations/Version202605141200002234_tao.php

```
@@ -0,0 +1,34 @@
+
+/**
+ * phpcs:disable Squiz.Classes.ValidClassName
+ */
+final class Version202605141200002234_tao extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Registers default translation bundle processor for build-time PO post-processing';
+    }
+
+    public function up(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)